cosmos/infra/azuredeploy_mongo.json

```diff
@@ -233,6 +233,34 @@
                 "description": "Specifies application insight connection string for live monitering"
             }
         },
+        "pointOperationLatencyThresholdInMS": {
```

Comment:
As the name suggests, this template is for mongo binding. There is no point in adding this to this template.

Reply:
The binding name azurecosmos made me think it is used for all cosmos APIs. I see now that mongo is using a different binding.

Removed the changes from this file. 
azurecosmos binding can be called azurecosmos-sql may be to avoid confusion.